lib/Dialect/AIE/Transforms/AIEObjectFifoStatefulTransform.cpp

```diff
@@ -748,8 +736,9 @@ struct AIEObjectFifoStatefulTransformPass
         succ = builder.createBlock(endBlock);
 
       builder.setInsertionPointToStart(curr);
+      int offset = 0;
       if (isDistribute || isJoin)
-        offset = extraOffset * bytes;
```

Comment:
Was this a functional error?  Do we have a test for the fix?

Reply:
It wasn't an error - it's just no longer correct when `offset` is in terms of # of element rather than #number of bytes